presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaMetadata.java

```diff
@@ -69,7 +69,6 @@
                     .column("column_default", VARCHAR)
                     .column("is_nullable", VARCHAR)
                     .column("data_type", VARCHAR)
-                    .column("is_partition_key", VARCHAR)
```

Comment:
the commit message should mention removing `is_partition_key` from information schema... maybe a separate commit?